src/components/shared/forms/editUserForm.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+
+import { User, ExpressValidationError } from 'devu-shared-modules'
+
+import { useAppDispatch } from 'redux/hooks'
+import { SET_ALERT } from 'redux/types/active.types'
+
+import TextField from 'components/shared/inputs/textField'
+
+import RequestService from 'services/request.service'
+
+type Props = {
+  user: User
+}
+
+const EditUserForm = ({ user }: Props) => {
+  const [formData, setFormData] = useState<User>(user)
+  const setAlert = useAppDispatch<typeof SET_ALERT>(SET_ALERT)
+
+  const handleUpdatePreferredName = (preferredName: string) => setFormData({ ...formData, preferredName })
+
+  const submit = () => {
```

Comment:
to follow the react convention, this should probably be called `handleSubmit` rather than just submit

Reply:
changed